feature/onboarding/welcome/src/main/kotlin/app/k9mail/feature/onboarding/welcome/ui/WelcomeContent.kt

```diff
@@ -125,6 +149,20 @@ private fun WelcomeTitle(
     }
 }
 
+@Composable
+private fun WelcomeMessageItem(
+){
```

Comment:
Please add the modifier here:

```suggestion
    modifier: Modifier = Modifier,
) {
```